package/appmarkable/package

```diff
@@ -1,6 +1,6 @@
 # vim: set ft=sh:
 pkgname=appmarkable
-pkgdesc="Turn your program into a very simple app for draft similar"
+pkgdesc="Turn your program into a very simple app for draft"
 url=https://github.com/LinusCDE/appmarkable
```

Comment:
Perhaps you should quote this `url=` field too for consistency with the other changes?